environments/common/inventory/group_vars/all/nfs.yml

```diff
@@ -16,8 +16,6 @@ nfs_configurations:
     nfs_export: "/exports/home" # assumes skeleton TF is being used
     nfs_client_mnt_point: "/home"
 
-  - comment: Export /exports/cluster from Slurm control node
-    nfs_enable:
-        server: "{{ inventory_hostname in groups['control'] }}"
-        clients: false
-    nfs_export: "/exports/cluster"
+# Set 'secure' to prevent tunneling nfs mounts
+# Cannot set 'root_squash' due to home directory creation
```

Comment:
Can we create the home directories on the node exporting the nfs share?

Reply:
Yes but it was agreed to do that as a a separate PR, it is quite a complex change c.f. this one.